recipes/logr/config.yml

```diff
@@ -0,0 +1,5 @@
+versions:
+  "0.1.0":
+    folder: 0.1.0
```

Comment:
```suggestion
```

Reply:
0.1.0 was already checked in... when version == folder name the config.yml is not required.... it's a sneaky features of c31